funny/funny.go

```diff
@@ -175,7 +175,7 @@ func (funny *Funny) ChatHandler(conv *slick.Conversation, msg *slick.Message) {
 
 	} else if msg.Text == "ls" {
 
-		conv.Reply(msg, "/code deploy/      Contributors-Guide/ image_server/     sheep_porn/     streambed/\nstreamhead/  README.md")
+		conv.Reply(msg, "/code deploy/      Contributors-Guide/ image_server/     streambed/\nstreamhead/  README.md")
```

Comment:
these jokes were plotly-related.. I think we can move the "funny" plugin to our internal repo, and tweak at will.. you'll then have read-write ! :)